the_dovepress_peacock: DovePress changed HTML structure, PDF links no longer found

## Problem

5 tests for `the_dovepress_peacock` are failing because DovePress changed their article page HTML structure. The function can no longer find either:
- Links matching `article/download/` in href
- A `citation_pdf_url` meta tag

```
FAILED tests/findit/test_dovepress.py::TestDovePressTest::test_dovepress_waltz_cmar_article
FAILED tests/findit/test_dovepress.py::TestDovePressTest::test_dovepress_waltz_ijin_article
FAILED tests/findit/test_dovepress.py::TestDovePressTest::test_dovepress_waltz_opth_article
FAILED tests/findit/test_dovepress.py::TestDovepressXMLFixtures::test_dovepress_xml_37822558_adolesc_health_med_ther
FAILED tests/findit/test_dovepress.py::TestDovepressXMLFixtures::test_dovepress_xml_35592492_adolesc_health_med_ther
```

All fail with: `NoPDFLink: MISSING: No PDF download link found in DovePress article HTML`

## Example request flow

DOI `10.2147/IJN.S420748` resolves to:
```
GET https://www.dovepress.com/plant-derived-exosome-like-nanovesicles-current-progress-and-prospects-peer-reviewed-fulltext-article-IJN  →  200 OK
```
The page returns 200 but the HTML no longer contains `article/download/` links or `citation_pdf_url` meta tags.

## Affected PMIDs / DOIs

| PMID | Journal | DOI |
|------|---------|-----|
| 37693885 | Int J Nanomedicine | 10.2147/IJN.S420748 |
| 37736107 | Clin Ophthalmol | 10.2147/OPTH.S392665 |
| 36873252 | Cancer Manag Res | 10.2147/CMAR.S400013 |
| 37822558 | Adolesc Health Med Ther | 10.2147/AHMT.S429238 |
| 35592492 | Adolesc Health Med Ther | 10.2147/AHMT.S358140 |

## Investigation needed

- Fetch the current HTML for one of these article URLs and identify where the PDF download link is now located
- Update `the_dovepress_peacock` parsing logic to handle the new HTML structure
- Update the `TestDovepressXMLFixtures` expected URLs if the URL format has also changed

--- a/setup.py
+++ b/setup.py
@@ -97,6 +97,7 @@ setup(
         "lxml",
         "lxml_html_clean",
         "requests",
+        "brotli",
         "habanero",
         "tabulate",
         "cssselect",

